docs/api.md

```diff
@@ -70,6 +70,39 @@ or add to your `~/.gdbinit`:
 $ echo source /path/to/newcmd.py >> ~/.gdbinit
 ```
 
+## Custom context panes ##
+
+Sometimes you want something similar to a command to run on each break-like event 
+and display itself as a part of the GEF context. Here is a simple example of how to
+make a custom context pane:
+```python
+__start_time__ = int(time.time())
+def wasted_time_debugging():
+    gef_print("You have wasted {} seconds!".format(int(time.time()) - __start_time__))
+
+def wasted_time_debugging_title():
```

Comment:
Now that I read this, it would make more sense to make `register_external_context_pane` a Python decorator which would allow to make something like:
```python
@register_external_context_pane("wasted_time_debugging")
def wasted_time_debugging_title():
```

Reply:
Discussed on Discord, resolving for now, might be done in the future.